uthdev/Quick-Credit
# #169062659 Refactor all endpoints

#### What does this PR do
Refactor all the endpoints
#### Description of Task to be completed?
 - add .eslintignore file
 - refactor all endpoints
 - add jsDoc documentation
 - changed eslint rules
 - add cors packages
#### How should this be manually tested?
- clone the repo
- run `npm install`
- Ensure you have your postgreSQL setup
- Run `npm test`
#### What are the relevant pivotal tracker stories?
[#169062659](https://www.pivotaltracker.com/story/show/169062659)
